CHANGELOG.md

```diff
@@ -5,7 +5,8 @@
 - Fix external syntax highlighting for embedded ocaml source in comments (#906)
 - Add `ocaml.repl.useUtop` setting to control whether to use Utop for the REPL
   (#911)
-- Fix the indentation values for simple comments and documentation comments (#182)
+- Fix the indentation values for simple comments and documentation comments
+  (#182)
```

Comment:
This should be the PR number, 928.

Reply:
Also, it's not really a "fix" since the issue was a lack of a feature and not a bug. I think a more accurate description would start with
> Add indentation rules for ...